cucumber-js/_setup_cucumber_runner.js

```diff
@@ -63,7 +63,14 @@ Before(function({pickle}) {
   }
 });
 
-After(async function() {
+After(async function(testCase) {
+
+  //send test-case result to cloud provider
+  const {result} = testCase;
+  if (this.client.transport && this.client.transport.usingBrowserstack && result) {
```

Comment:
This is too specific. We need a default method on the base transport and then overwrite that in the browserstack transport.